マニュアル.md

```diff
@@ -1,51 +1,203 @@
-**※このマニュアルは、日本語が母国語ではない人が書いたものです。 文法的な誤りは山ほどあるかもしれません。 このページを編集したいと思う方はプルリクエストを作成して下さい。 よろしくお願いします。**
-
 # BepInEx LoveMachine
+[![.NET](https://github.com/Sauceke/LoveMachine/actions/workflows/commit.yml/badge.svg)](#)
+[![Download](https://img.shields.io/github/downloads/Sauceke/LoveMachine/total)][installer]
+[![Patreon](https://shields.io/badge/patreon-grey?logo=patreon)][Patreon]
 
 | [インストーラ](https://github.com/Sauceke/LoveMachine/releases/latest/download/LoveMachineInstaller.exe) |
 |----|
 
-LoveMachineは、ゲームエンジンUnityを使った3Dエロゲを連動オナホールやバイブレータに対応できるようにする為のプラグインです。
-
-現在互換性があるゲームは：
-* **コイカツ！（VRも）**
-* **ハニーセレクト２（VRも）**
-* **コイカツ！サンシャイン（VRも）**
-* **プレイホーム（VRも）**
-* **奉課後輪姦中毒（VRも、[AGHVR](https://github.com/Eusth/AGHVR)で）**
-* **Our Apartment**
-* **インサルトオーダー（VRも、[IOVR](https://github.com/Eusth/IOVR)で）**
-* **カスタムオーダーメイド3D2（VRも）**
-* **ROOMガール**
-* **VRカノジョ**
-* **聖騎士リッカの物語**
-* **セクロスフィア**
-* **Last Evil**
-* **サキュバス カフェ**
-
-LoveMachineは[buttplug](https://buttplug.io/)というオープンソースプロトコルのお陰でいろんなデバイスと繋がることが出来ます。
-
-現在互換性があるデバイスは：
+LoveMachineはアダルトゲームのキャラクターと連動してアダルトグッズを動かすプラグインです。 
+
+[一部の自律型デバイス](#supported-devices)について、以下のゲームに対応しています。
+| ゲームタイトル | 開発者 | VR対応 |
+|:----|:----|:----|
+| [カスタムオーダーメイド3D2] | Kiss | ⭕ |
+| [聖騎士リッカの物語] | もぐらソフト | ❌ |
+| [ハニーセレクト2] | ILLUSION | ⭕ |
+| [放課後輪姦中毒] | みこにそみ | ⭕※[AGHVR]のみ |
+| [インサルトオーダー] | みこにそみ | ⭕※[IOVR]のみ |
+| [コイカツ！] | ILLUSION | ⭕ |
+| [コイカツ！パーティ] | ILLUSION | ⭕ |
+| [コイカツ！サンシャイン] | ILLUSION | ⭕ |
+| [Last Evil] | Flametorch | ❌ |
+| [Our Apartment] | Momoiro Software | ❌ |
+| [プレイホーム] | ILLUSION | ⭕ |
+| [ROOMガール] | ILLUSION | ❌ |
+| [セクロスフィア] | ILLUSION | ❌ |
+| [Succubus Cafe] | MIGI STUDIO | ❌ |
+| [VRカノジョ] | ILLUSION | ⭕ |
+
+## サポートしているデバイス
+LoveMachineはアダルトグッズとの連動を可能にする[Buttplug.io]プロジェクトに頼って作られています。執筆時点でButtplug.ioは200以上のデバイスをサポートしています。
+
+このプラグインは**上下運動**、**振動**、**回転**、**締め付け**機能に対応しています。
+
+以下は実際にMODでテストしたデバイスの一覧です：
+
+電動オナホ
+- [The Handy]
+- OSR2
+- KIIROO KEON
+
+振動機能付きオナホ
+- [Lovense Gush]
+- [Lovense Max 2]
+- [Lovense Diamo]
+- [Lovense Domi 2]
+- [Lovense Calor]
+- The Xbox gamepad
+
+回転機能付きオナホ
+- Vorze A10 Cyclone
+
+[LoveMachine.Experiments]プラグインは、[コイカツ！]と[コイカツ！サンシャイン]で実験的な2つの深度センシングデバイス[Lovense Calor]と[Hotdog]のサポートを追加します。
+
+## インストール
+Download and run the [installer]. If you encounter the "Windows protected your PC" message, click More info > Run anyway.

```

Comment:
Do you need help in translating this part?
I don't have access to a Windows machine right now, but this is what the message looks like in English: https://i.stack.imgur.com/3t0tg.jpg